test/problem1_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rspec'
+require_relative '../src/problem1'
+
+describe 'Problem1' do
+
+  array = [2,1,3,3,4]
```

Comment:
This is not the best way to write this variables,

Consider using [let](https://relishapp.com/rspec/rspec-core/docs/helper-methods/let-and-let)